cli/index.ts

```diff
@@ -149,6 +149,35 @@ const jsonOutputOption: INamedOption<yargs.Options> = {
   }
 };
 
+const varsOption: INamedOption<yargs.Options> = {
+  name: "vars",
+  option: {
+    describe:
+      "Variables to inject that are exposed at compile and runtime via `dataform.projectConfig.vars.myVariableName`.\n" +
+      "They should take the structure of '--vars=someKey:someValue,someOtherKey:someOtherValue'.",
```

Comment:
the newline in the middle of this description is a bit odd!

also i do think we should use `=` rather than `:`